apps/oc_chef_wm/itest/oc_chef_wm_policies_SUITE.erl

```diff
@@ -0,0 +1,283 @@
+%% -*- erlang-indent-level: 4;indent-tabs-mode: nil; fill-column: 92 -*-
+%% ex: ts=4 sw=4 et
+%% @author Stephen Delano <stephen@opscode.com>
+%% Copyright 2013 Opscode, Inc. All Rights Reserved.
+
+-module(oc_chef_wm_policies_SUITE).
+
+-include_lib("common_test/include/ct.hrl").
+-include("../../../include/chef_types.hrl").
+-include("../../../include/oc_chef_types.hrl").
+-include_lib("eunit/include/eunit.hrl").
+
+-record(context, {reqid :: binary(),
+                  otto_connection,
+                  darklaunch = undefined}).
+
+-compile([export_all, {parse_transform, lager_transform}]).
+
+-define(ORG_AUTHZ_ID, <<"10000000000000000000000000000002">>).
+-define(AUTHZ_ID, <<"00000000000000000000000000000003">>).
+-define(CLIENT_NAME, <<"test-client">>).
+-define(ORG_NAME, <<"org1">>).
+
+init_per_suite(Config) ->
+    Config2 = setup_helper:start_server(Config),
+
+    OrganizationRecord = chef_object:new_record(oc_chef_organization,
+                                                nil,
+                                                ?ORG_AUTHZ_ID,
+                                                {[{<<"name">>, ?ORG_NAME},
+                                                  {<<"full_name">>, ?ORG_NAME}]}),
+    Result2 = chef_db:create(OrganizationRecord,
+                   #context{reqid = <<"fake-req-id">>},
```

Comment:
`chef_db:make_context(<<"fake-req-id">>)` will take care of this so we don't have to worry about the record itself.